CONTRIBUTE.md

```diff
@@ -1,59 +1,72 @@
+ 8. Make sure that the libraries install successfully and the unit tests pass 
+    on Travis-CI.
+ 9. Make sure that the changes come with documentation, e.g. internally in the 
+    C code and through Python docstrings. Make sure that example code in the 
+    `examples/` directory has been updated appropriately.
 
 Things to do if you are adding new features to the CCL C lib
 ------------------------------------------------------------
- 1. If you haven't created a new files (i.e. you only edited
-    an existing one), you don't need to do any of this.
- 2. When adding a new source file (.c), put it in src. The
-    the new file should be listed under libccl_la_SOURCES in
-    Makefile.am.
- 3. When adding a new header file (.h), put it in include.
-    The new file should be listed under include_HEADERS in
-    include/Makefile.am. It should also have doxygen
-    compatible documentation.
- 4. When adding new unit test files, they should be listed
-    under check_ccl_SOURCES in Makefile.am
- 5. Any other new files that should be included with the
-    library (e.g. documentation files) should be listed
-    under EXTRA_DIST in Makefile.am.
- 6. The file configure.ac may have to be modified when 
-    adding new features (e.g. adding a new dependency), but
-    not too often.
- 7. After making changes to any of these files, developers
-    should run:
-      $> autoreconf -i
-    and commit any modified files (some files will be
-    automatically modified by this command).
- 8. Any updates to CCL should be documented in the CCL note
-    that you can find in the directory doc/0000-ccl_note.
-    You will need to edit main.tex, compile with: $> make
-    and commit your changes, including the updated pdf.
-
-More autotools fun:
+ 1. If you haven't created a new files (i.e. you only edited an existing one), 
+    you don't need to do any of this.
+ 2. When adding a new source file (.c), put it in `src/`. The new file should 
+    be listed under `libccl_la_SOURCES` in `Makefile.am`.
+ 3. When adding a new header file (.h), put it in `include/`. The new file 
+    should be listed under `include_HEADERS` in `include/Makefile.am`. It 
+    should also have `doxygen`-compatible documentation strings.
+ 4. When adding new unit test files, they should be listed under 
+    `check_ccl_SOURCES` in `Makefile.am`.
+ 5. Any other new files that should be included with the library (e.g. 
+    documentation files) should be listed under `EXTRA_DIST` in `Makefile.am`.
+ 6. The file `configure.ac` may have to be modified when adding new features 
+    (e.g. adding a new dependency).
+ 7. After making changes to any of these files, developers should run `autoreconf -i`.
+    and commit any modified files (some files will be automatically regenerated 
+    by this command).
+ 8. Make sure that you regenerate the SWIG Python wrapper (see below).
+ 9. Any updates to CCL should be documented in the CCL note that you can find 
```

Comment:
I think this should also be something to check for in the above list, when reviewing a PR.